docs/source/performance.md

```diff
@@ -126,84 +124,82 @@ Orderer: &OrdererDefaults
 
 ## Application considerations
 
-When designing the application architecture, decisions can affect the overall performance of the network and the application. Here we cover some of the considerations.
+When designing your application architecture, various choices can affect the overall performance of both the application and the network. These application considerations are described below.
 
-### Avoid CouchDB for high throughput applications
+### Avoid CouchDB for high-throughput applications
 
-CouchDB performance is noticably slower than embedded LevelDB, sometimes by a factor of 2x slower. The only additional capability that a CouchDB state database provides is JSON (rich) queries as stated in the [Fabric state database documentation](./deploypeer/peerplan.html#state-database). You should also never allow direct access to the CouchDB data (for instance via Fauxton UI) to ensure the integrity of the state data.
+CouchDB performance is noticably slower than embedded LevelDB, sometimes by a factor of 2x slower. The only additional capability that a CouchDB state database provides is JSON (rich) queries, as stated in the [Fabric state database documentation](./deploypeer/peerplan.html#state-database). In addition, to ensure the integrity of the state data you must never allow direct access to CouchDB data (for instance via Fauxton UI). 
 
-CouchDB as a state database also has other limitations which will have impacts on performance and requires additional hardware resource (and cost). Additionally, JSON queries are not re-executed at validation time and thus there is no built-in protection from phantom reads as there is with range queries (your application must be designed to tolerate phantom reads when using JSON queries, for example when records are added to state between the time of chaincode execution and block validation). For these reasons, consider using range queries based on additional keys rather than using CouchDB JSON with queries.
+CouchDB as a state database has other limitations which reduce overall network performance and require additional hardware resource (and cost). For one, JSON queries are not re-executed at validation time, so CouchDB does not offer built-in protection from phantom reads. Such protection is provided, however, by range queries. (Your application must be designed to tolerate phantom reads when using JSON queries, such as when records are added to state between the times of chaincode execution and block validation). You should therefore consider using range queries based on additional keys for high-throughput applications, rather than using CouchDB JSON with queries.
```

Comment:
```so CouchDB does not offer built-in protection from phantom reads```
It's not CouchDB that offers this protection, it's fabric that offers this and can't do it for JSON queries. The above reads that it's more a CouchDB limitation

Reply:
fixed both.